packages/xo-server-auth-saml/src/index.js

```diff
@@ -24,7 +24,10 @@ export const configurationSchema = {
     },
     usernameField: {
       title: 'Username field',
-      description: 'Field to use as the XO username',
+      description: `Field to use as the XO username
+
+You should try \`http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress\` if you are using Microsoft Azure Active Directory.
```

Comment:
When I Visit this resource I got this message: `The resource you are looking for has been removed, had its name changed, or is temporarily unavailable`

Reply:
This URL does not need to be accessible, it's only a key to use in some configuration.